test_suite.sh

```diff
@@ -0,0 +1,228 @@
+#!/bin/bash
+#
+# Run sanity check for all requested modules (and built dependencies)?
+# get ec from diff
+# set up EasyBuild
+# run `eb --sanity-only ec`
+
+display_help() {
+  echo "usage: $0 [OPTIONS]"
+  echo "  -g | --generic         -  instructs script to test for generic architecture target"
+  echo "  -h | --help            -  display this usage information"
+  echo "  -x | --http-proxy URL  -  provides URL for the environment variable http_proxy"
+  echo "  -y | --https-proxy URL -  provides URL for the environment variable https_proxy"
+}
+
+POSITIONAL_ARGS=()
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -g|--generic)
+      EASYBUILD_OPTARCH="GENERIC"
+      shift
+      ;;
+    -h|--help)
+      display_help  # Call your function
+      # no shifting needed here, we're done.
+      exit 0
+      ;;
+    -x|--http-proxy)
+      export http_proxy="$2"
+      shift 2
+      ;;
+    -y|--https-proxy)
```

Comment:
again lots of copy-pasting, we need a common script to handle options?

Reply:
Agree, but again, let's consider this in a follow-up PR